tests/unit_tests/test_media/test_media_logging.py

```diff
@@ -0,0 +1,57 @@
+from pathlib import Path
+
+import numpy as np
+
+data = np.random.randint(255, size=(1000))
+
+
+def test_log_media_saves_to_run_directory(
```

Comment:
alternatively you could just parametrize this function with the different media types

Reply:
I like use parametrize here! will make that change